dkms.in

```diff
@@ -643,6 +643,15 @@ read_conf()
     [[ $BUILD_EXCLUSIVE_KERNEL_MIN && "$(VER "$1")" < "$(VER "$BUILD_EXCLUSIVE_KERNEL_MIN")" ]] && build_exclude="yes"
     [[ $BUILD_EXCLUSIVE_KERNEL_MAX && "$(VER "$1")" > "$(VER "$BUILD_EXCLUSIVE_KERNEL_MAX")" ]] && build_exclude="yes"
     [[ $BUILD_EXCLUSIVE_ARCH && ! $2 =~ $BUILD_EXCLUSIVE_ARCH ]] && build_exclude="yes"
+    if [[ $BUILD_EXCLUSIVE_CONFIG && -e "${kernel_config}" ]]; then
+        local kconf
+        for kconf in $BUILD_EXCLUSIVE_CONFIG ; do
+            case "$kconf" in
+                !*) grep -q -E "^${kconf#!}=[ym]" "${kernel_config}" && build_exclude="yes" ;;
+                *)  grep -q -E "^${kconf}=[ym]" "${kernel_config}" || build_exclude="yes" ;;
```

Comment:
We don't need the `-E` here, do we? This seems to work with basic (the default), perl and extended RE. 

Ideally this feature will come some tests, although I can understand if you feel like not writing any more (for the time being at least) . Opened a task for the future https://github.com/dell/dkms/issues/327

Reply:
No, `-E` is not needed. I probably started with a more complicated expression ...